### PR TITLE
spelling fixes (just comments)

### PR DESCRIPTION
**Pull request by ka7**

### CODE REVIEW

File: src/Boot/Windows/BootMain.cpp [hunk 1]
```diff
@@ -610,7 +610,7 @@ static void BootMenu ()
 
 				if (result == BiosResultSuccess && *(uint16 *) (SectorBuffer + TC_LB_SIZE - 2) == 0xaa55)
 				{
-					// Windows writes boot loader on all NTFS/FAT filesytems it creates and, therefore,
+					// Windows writes boot loader on all NTFS/FAT filesystems it creates and, therefore,
```
Comment (frank-dittrich) [inline src/Boot/Windows/BootMain.cpp]: s/filesystems/file systems/

File: src/Common/Inflate.c [hunk 2]
```diff
@@ -169,7 +169,7 @@
       end-of-block.  Note however that the static length tree defines
       288 codes just to fill out the Huffman codes.  Codes 286 and 287
       cannot be used though, since there is no length base or extra bits
-      defined for them.  Similarily, there are up to 30 distance codes.
```
Comment (Diapolo) [inline src/Common/Inflate.c]: IMHO you could also remove that space before :)?

**Comment by Diapolo** [comment]
Seems like a nice and quick pull.

File: src/Crypto/Aes_x64.asm [hunk 3]
```diff
@@ -35,7 +35,7 @@
 ;
 
 ; An AES implementation for AMD64 processors using the YASM assembler.  This
-; implemetation provides only encryption, decryption and hence requires key
```
Comment (Diapolo) [inline src/Crypto/Aes_x64.asm]: Same at the beginning of this sentence like my comment above.

File: src/Common/Dlgcode.c [hunk 4]
```diff
@@ -2545,7 +2545,7 @@ static void LoadSystemDll (LPCTSTR szModuleName, HMODULE *pHandle, BOOL bIgnoreE
 {
 	wchar_t dllPath[MAX_PATH];
 
-	/* Load dll explictely from System32 to avoid Dll hijacking attacks*/
+	/* Load dll explicitely from System32 to avoid Dll hijacking attacks*/
```
Comment (soroshsabz) [inline src/Common/Dlgcode.c:2548]: ITNOA

I think **_explicitly_** is correct

File: src/ExpandVolume/WinMain.cpp [hunk 5]
```diff
@@ -445,7 +445,7 @@ BOOL CALLBACK ExtcvPasswordDlgProc (HWND hwndDlg, UINT msg, WPARAM wParam, LPARA
 
 			SetPim (hwndDlg, IDC_PIM, *pim);
 
-			/* make PIM field visible if a PIM value has been explicitely specified */
+			/* make PIM field visible if a PIM value has been explicitly specified */
```
Comment (soroshsabz) [inline src/ExpandVolume/WinMain.cpp:448]: I think explicitly is correct

**Comment by soroshsabz** [comment]
@ka7 Please pull from master again, to resolve conflict files.

**Pull request closed without merging**